x-pack/test_serverless/functional/test_suites/observability/cases/view_case.ts

```diff
@@ -0,0 +1,906 @@
+import { v4 as uuidv4 } from 'uuid';
+import { CaseSeverity, CaseStatuses } from '@kbn/cases-plugin/common/types/domain';
+import { setTimeout as setTimeoutAsync } from 'timers/promises';
+
+import { OBSERVABILITY_OWNER } from '@kbn/cases-plugin/common';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import {
+  createOneCaseBeforeDeleteAllAfter,
+  createOneCaseBeforeEachDeleteAllAfterEach,
+  createAndNavigateToCase,
+} from '../../../../shared/lib/cases/helpers';
+
+const owner = OBSERVABILITY_OWNER;
+
+export default ({ getPageObject, getService }: FtrProviderContext) => {
+  const header = getPageObject('header');
+  const testSubjects = getService('testSubjects');
+  const cases = getService('cases');
+  const find = getService('find');
+
+  const retry = getService('retry');
+  const comboBox = getService('comboBox');
+  const browser = getService('browser');
+  const svlCommonNavigation = getPageObject('svlCommonNavigation');
+
+  describe('View case', () => {
+    describe('page', () => {
+      createOneCaseBeforeDeleteAllAfter(getPageObject, getService, owner);
+
+      it('should show the case view page correctly', async () => {
+        await testSubjects.existOrFail('case-view-title');
+        await testSubjects.existOrFail('header-page-supplements');
+
+        await testSubjects.existOrFail('case-view-tab-title-activity');
+        await testSubjects.existOrFail('case-view-tab-title-files');
+        await testSubjects.existOrFail('description');
+
+        await testSubjects.existOrFail('case-view-activity');
+
+        await testSubjects.existOrFail('case-view-assignees');
+        await testSubjects.existOrFail('sidebar-severity');
+        await testSubjects.existOrFail('case-view-user-list-reporter');
+        await testSubjects.existOrFail('case-view-user-list-participants');
+        await testSubjects.existOrFail('case-view-tag-list');
+        await testSubjects.existOrFail('cases-categories');
+        await testSubjects.existOrFail('sidebar-connectors');
+      });
+    });
+
+    describe('properties', () => {
+      createOneCaseBeforeDeleteAllAfter(getPageObject, getService, owner);
+
+      it('edits a case title from the case view page', async () => {
+        const newTitle = `test-${uuidv4()}`;
+
+        await testSubjects.click('editable-title-header-value');
+        await testSubjects.setValue('editable-title-input-field', newTitle);
+        await testSubjects.click('editable-title-submit-btn');
+
+        // wait for backend response
+        await retry.tryForTime(5000, async () => {
+          const title = await find.byCssSelector('[data-test-subj="editable-title-header-value"]');
+          expect(await title.getVisibleText()).equal(newTitle);
+        });
+
+        // validate user action
+        await find.byCssSelector('[data-test-subj*="title-update-action"]');
+      });
+
+      it('shows error message when title is more than 160 characters', async () => {
```

Comment:
As per discussion in yesterday's meeting, shall we remove validations check from serverless tests?

Reply:
Removed a bunch of them and left some smoke tests only.